app/src/main/java/no/usn/mob3000/domain/usecase/content/docs/FetchDocUseCase.kt

```diff
@@ -12,9 +12,13 @@ import no.usn.mob3000.domain.model.content.DocsData
  * @created 2024-10-30
  */
 class FetchDocUseCase(
-    private val fetchRepository: DocsRepository = DocsRepository()
+    private val fetchRepository: DocsRepository
 ) {
-    suspend fun fetchDocumentations(): Result<List<DocsData>> {
-        return fetchRepository.fetchDocuments()
+    suspend fun fetchLocalDocs(): Result<List<DocsData>> {
+        return fetchRepository.fetchAllDocsLocal()
+    }
+
+    suspend fun refreshRoomFromNetwork(): Result<Unit> {
+        return fetchRepository.refreshDocsFromNetwork()
```

Comment:
This should have been in its own use case since it's not a private function in use in that use case.